cpp/demo/interpolation_different_meshes/main.cpp

```diff
@@ -72,11 +73,18 @@ int main(int argc, char* argv[])
     u_hex->interpolate(*u_tet, nmm_interpolation_data);
 
 #ifdef HAS_ADIOS2
-    io::VTXWriter write_tet(mesh_tet->comm(), "u_tet.vtx", {u_tet});
+    io::VTXWriter<double> write_tet(mesh_tet->comm(), "u_tet.vtx", {u_tet});
     write_tet.write(0.0);
 
-    io::VTXWriter write_hex(mesh_hex->comm(), "u_hex.vtx", {u_hex});
+    io::VTXWriter<double> write_hex(mesh_hex->comm(), "u_hex.vtx", {u_hex});
```

Comment:
Have you tested that we can open `VTXWriter` files written in single precision with Paraview.
Also, the `vtx` file-extension should be `".bp"` not `".vtx"`

Reply:
Tested in a different demo. Extension fixed.